manageiq-operator/pkg/helpers/miq-components/orchestrator.go

```diff
@@ -204,3 +204,12 @@ func NewOrchestratorDeployment(cr *miqv1alpha1.ManageIQ) (*appsv1.Deployment, er
 
 	return deployment, nil
 }
+
+func dbRegionCode(cr *miqv1alpha1.ManageIQ) string {
+	regionCode := "0"
+	if cr.Spec.DatabaseRegion != "" {
+		regionCode = cr.Spec.DatabaseRegion
+	}
+
+	return regionCode
+}
```

Comment:
This logic is already present in https://github.com/ManageIQ/manageiq-pods/blob/2914b29a0c4a9aedb13cc3c0060d19ad48008fbf/manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go#L215-L217. No need to duplicate it here.